docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,520 @@
+```
+
+You should back up this file to a safe location that can be retrieved later.
+
+To reuse the wallet for subsequent redeployments or share the RCU tables between different domains:
+
+1. Store the wallet in a secret:
+
+```
+    opss_wallet_util.sh -r [-wf <name of the wallet file. Default ./ewallet.p12>] [-ws <name of the secret. Default DOMAIN_UID-opss-walletfile-secret> ]
+
+```
+
+2. Modify the domain resource YAML file to provide the secret names:
+
+```
+  configuration:
+    opss:
+      # Name of secret with walletPassword for extracting the wallet
+      walletPasswordSecret: sample-domain1-opss-wallet-password-secret      
+      # Name of secret with walletFile containing base64 encoded opss wallet
+      walletFileSecret: sample-domain1-opss-walletfile-secret
+
+```
+
+See [Reusing an RCU database]({{< relref "/userguide/managing-domains/model-in-image/reusing-rcu.md" >}}) for instructions.
+
+### Use the WebLogic Image Tool to create an image
+
+Model in Image must contain a WebLogic install, a WebLogic Deploy Tool install, and your WDT model files. You can use the sample `./build.sh` script to build this image, which will perform the following steps for you:
+
+  - Uses `docker pull` to obtain a base image. (See [Prerequisites for all domain types](#prerequisites-for-all-domain-types) to set up access to the base image.)
+  - Downloads the latest WebLogic Image Tool and WebLogic Deploy Tool to `WORKDIR`.
+  - Creates and populates a staging directory `$WORKDIR/models` that contains your WDT model files and WDT application archive.
+    - Builds  a simple servlet application in `$SAMPLEDIR/sample_app` into a WDT model application archive `$WORKDIR/models/archive1.zip`.
+    - Copies sample model files from `$SAMPLEDIR/model-in-image` to `$WORKDIR/models`. This uses a model file that is appropriate to the domain type (for example, the `JRF` domain model includes database access configuration).
+  - Uses the WebLogic Image Tool and the `$WORKDIR/models` staging directory to create a final image named `model-in-image:v1` that layers on the base image. Specifically, it runs the WebLogic Image Tool with its `update` option, which:
+    - Builds the final image as a layer on the base image.
+    - Puts a WDT install in image location, `/u01/wdt/weblogic-deploy`.
+    - Copies the WDT model, properties, and application archive from `$WORDKIR/models` to image location, `/u01/wdt/models`.
+
+The script expects `JAVA_HOME`, `WDT_DOMAIN_TYPE`, and `WORKDIR` to already be initialized. (See [Prerequisites for all domain types](#prerequisites-for-all-domain-types).)
+
+Run the script:
+
+  ```
+  $SAMPLEDIR/build.sh
+  ```
+
+If you intend to use a remote Docker registry, you need to tag and push the image to the remote Docker registry.
+
+1.  Tag the image for the remote Docker registry, for example:
+
+```
+docker tag <image-name>:<tag> <region-key>.ocir.io/<tenancy-namespace>/<repo-name>/<image-name>:<tag>
```

Comment:
These instructions are to push the image to a registry in OCI why?  Other samples in domain-home-in-image call the image "domain-home-in-image:12.2.1.3" we should call our image a generic name "domain-model-in-image:12.2.1.4"

Reply:
Tom AI: Change "docker tag <image-name>:<tag> <region-key>.ocir.io/<tenancy-namespace>/<repo-name>/<image-name>:<tag>" to "docker tag model-in-image:v1 my.remote.registry.com/model-in-image:v1".   Change below as well...", plus change below accordingly.

Reply:
Done.